client/app/bundles/course/achievement/badge-uploader.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { PropTypes } from 'react';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import Avatar from 'material-ui/Avatar';
+import Badge from 'material-ui/Badge';
+import Dropzone from 'react-dropzone';
+import IconButton from 'material-ui/IconButton';
+import CloseIcon from 'material-ui/svg-icons/navigation/close';
+
+const styles = {
+  badge: {
+    height: '120px',
+    marginRight: '1em',
+    width: '120px',
+  },
+  badgeIconActive: {
+    backgroundColor: 'rgb(200,200,200)',
```

Comment:
For standardisation, use [material ui colours](http://www.material-ui.com/#/customization/colors) instead. 

```
import { blue300 } from 'material-ui/styles/colors';
...
    backgroundColor: blue300,
```